.golangci.yaml

```diff
@@ -84,59 +84,37 @@ linters:
 linters-settings:
   gocritic:
     enabled-checks:
-      - appendAssign
       - appendCombine
-      - argOrder
```

Comment:
all of these removed checks are enabled by default and do not need to be explicitly enabled here